src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -52,6 +52,11 @@ class ModuleRepository extends AbstractObjectModelRepository
      */
     private $activeModulesPaths;
 
+    /**
+     * @var array
```

Comment:
Not blocking for this:
```suggestion
     * @var array<string, string> File paths indexed by module name
```